### DISCUSSION TITLE
"Join the waitlist" even though I'm in the private beta

### DISCUSSION THREAD

**Discussion by simjnd** [community/community#7194] [Copilot Conversations]
Hi everyone,

I received an email yesterday telling me I was (finally) in the private beta for Copilot. I tried a couple minutes ago installing the extension for VS Code (1.61.2) from the .vsix, generated a new Personal Access Token for my account and pasted it in Copilot's prompt for the token. A little popup showed up telling me I wasn't in the beta and I should sign up for it by joigning the waitlist.

What should I do? I tried uninstalling and re-installing it but it no longer even prompts me for my account token.

Happy to provide more information if necessary!

**Comment by davecheney** [comment]
I'm sorry you're having problems, can you please follow the installation instructions here https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/gettingstarted.md#1-installing-the-visual-studio-code-extension.
